package.json

```diff
@@ -15,7 +16,6 @@
     "babel-brunch": "~6.0.0",
     "uglify-js-brunch": "~2.0.1"
   },
-  "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "web/static/js/phoenix.js"],
```

Comment:
@josevalim That concern is imagined, IMO, but fine. This is the last npm related issue I'll bring to your attention, if at all possible.